broker/src/main/java/io/camunda/zeebe/broker/system/SystemContext.java

```diff
@@ -192,6 +194,20 @@ private void validateBackupCfg(final BackupStoreCfg backup) {
         LOG.warn(
             "Access credentials (accessKey, secretKey) not configured for S3 backup store. Credentials will be determined from environment (see https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html#credentials-chain)");
       }
+      // Create a throw away S3BackupStore to verify if all configurations are available
+      final S3BackupConfig storeConfig =
+          S3BackupConfig.from(
+              s3Config.getBucketName(),
+              s3Config.getEndpoint(),
+              s3Config.getRegion(),
+              s3Config.getAccessKey(),
+              s3Config.getSecretKey());
+      try {
```

Comment:
:wrench: Could make the backup store an `AutoCloseable` (with default implementation being `closeAsync().join()`, and then you can leverage the `try-with-resources`.

Reply:
I moved the validation to `S3BackupStore`. There it validates by only creating the client.